### ISSUE TITLE
CNV Panel WDL workflow input should no longer ingest tsv tables to parse bam file names.

### ISSUE THREAD

**Issue by LeeTL1220** [broadinstitute/gatk#3460]
Generally, it is easier on users (and people doing deployment) to have multiple files that list the bam and index files in corresponding order.  

So two files, rather than one:
```
sample1.bam
sample2.bam
....
```
and
```
sample1.bai
sample2.bai
....
```

For example, changing the input parameters to:

```
workflow CNVSomaticPanelWorkflow {
    # Workflow input files
    ....snip....
    # The next two parameters are files that list (in corresponding order) the bam files and bam index files, respectively.
    File normal_bams
    File normal_bam_idxs
    
    # Create (bam, bai) pairs for iterating over scatter loop.
    Array[Pair[File,File]] normal_pairs = zip(read_lines(normal_bams), read_lines(normal_bam_idxs))
    ....snip....
```

and further down in that file:

```
    ....snip....
    scatter (normal_pair in normal_pairs) {
        call CollectCoverage {
            input:
                padded_targets = select_first([PadTargets.padded_targets, ""]),
                bam = normal_pair.left,
                bam_idx = normal_pair.right,
                ref_fasta = ref_fasta,
                ref_fasta_fai = ref_fasta_fai,
                ref_fasta_dict = ref_fasta_dict,
                gatk_jar = gatk_jar,
                gatk_docker = gatk_docker
        }
    }
    ....snip....
```

**Comment by samuelklee** [comment]
This will be done after Cromwell 30 and Firecloud testing is complete.  We'll incorporate whatever changes were necessary to get those to pass as well as make the changes listed in #3940.

**Comment by samuelklee** [comment]
Closed in #3985.
